controller/getArticle.controller.js

```diff
@@ -0,0 +1,9 @@
+const { fetchArtcileByiD } = require("../model/fetchArticle.model");
```

Comment:
You have a spelling error in Article here - please can we correct this and all the other relevant ones :)